RELEASE-NOTES.md

```diff
@@ -9,669 +12,344 @@
-- User records have been modified so that an system-wide unique email
-  address is required to create a user.
-- User creation requests are serialized through `stanchion` to be
-  certain the email address is unique.
-- Bucket creation and deletion requests are serialized through
-  `stanchion` to ensure bucket names are unique in the system.
-- The `stanchion` serialization service is now required to be installed
-  and running for the system to be fully operational.
-- The concept of an administrative user has been added to the system. The credentials of the
-  administrative user must be added to the app.config files for `moss` and `stanchion`.
-- User credentials are now created using a url-safe base64 encoding module.
+#### Items hidden out in 2.0 by default
 
-## Known Issues
+The default value is not changed.
 
-- Object-level access control lists have not yet been implemented.
+| 1.5                                   | 2.0                        |
+|:--------------------------------------|:---------------------------|
+|`{fold_objects_for_list_keys, true}`   |`fold_objects_for_list_keys`|
+|`{n_val_1_get_requests, true}`         |`n_val_1_get_requests`      |
+|`{gc_paginated_indexes, true},`        |`gc.pagenated_indexes`      |
 
-# Riak CS 0.0.3 Release Notes
+#### Items commented out in 2.0 by default
 
-## Bugs Fixed
+Thus all of them are undefined and disabled, except modules. `rewrite_module` and
+`auth_module` will be commented out but the default value won't change from 1.5.
+This is for indicating operators how to change these to OOS API.
 
-- URL decode keys on put so they are represented correctly. This
-  eliminates confusion when objects with spaces in their names are
-  listed and when attempting to access them.
-- Properly handle zero-byte files
-- Reap all processes during file puts
+| 1.5                                   | 2.0                        |
+|:--------------------------------------|:---------------------------|
+|`{rewrite_module, riak_cs_s3_rewrite }`|`rewrite_module`            |
+|`{auth_module, riak_cs_s3_auth },`     |`auth_module`               |
+|`{admin_ip, "127.0.0.1"}`              |`admin.ip`                  |
+|`{admin_port, 8000 }`                  |`admin.port`                |
+|`{ssl, [`                              |                            |
+|`  {certfile, "./etc/cert.pem"}`       |`ssl.certfile`              |
+|`  {keyfile, "./etc/key.pem"}`         |`ssl.keyfile`               |
+
+#### `webmachine` section
 
-## Other Additions
+               {log_handlers, [
+                               {webmachine_log_handler, ["./log"]},
+                               {riak_cs_access_log_handler, []}
+                              ]}
+              ]},
 
-- Support for the s3cmd subcommands sync, du, and rb
+They will be defined in to `log.access.dir` and the output directory will
+just replaced. **TODO** we have to make kill switch for webmachine access logs.
```

Comment:
in `riak_cs.schema` 

```
{translation,
 "webmachine.log_handlers",
 fun(Conf) ->
   Handler = case cuttlefish:conf_get("log.access.dir", Conf) of
     "" -> [];
     Dir -> [{webmachine_access_log_handler, [Dir]}]
   end,
   Handler ++ [{riak_cs_access_log_handler, []}]
 end
}.
```

I guess leaving `log.access.dir` blank make access log disabled, right?